Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
This entire file should be removed from Git if it is autocreated by python

Reply:
I disagree on this one, this file locks libraries versions. In this file should be defined what versions of those libraries are we using so we can ensure compatibility.

Reply:
The versions are locked in Pipfile (without extension) file, there's no problem with that, this will be self-generated when the integrator executes `pipenv install` command